### PR TITLE
Updated homepage to have google crowdsourced map of relief centers

### LINKED ISSUE #500: Google Maps Relief Resources Page Link
### Subject of the issue
Currently no way to access google relief resources page link

[https://www.google.com/maps/d/u/0/viewer?mid=19pdXYBAk8RyaMjazX7mjJIJ9EqAyoRs5&ll=10.58818077013663%2C76.59135823100587&z=8](url)

### Your environment
Python3
Mozilla

### Steps to reproduce
NA

### Expected behaviour
Ideally, an ICON in the homepage directing towards the google maps page

### PR DESCRIPTION
**Pull request by ssat335**
### Issue Reference
This PR addresses the Issue : Fixes #500 #421 #512

### Summarize
Please, describe what the PR does, the changes you have made.

> Also, mention the key points if any to look into while code reviews
Just changed homepage which directs to the link. I agree with #421 may just start with crowdsourced data.

screenshot from 2018-08-18 15-08-21
screenshot from 2018-08-18 15-08-33

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -75,6 +75,13 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
           <span class="ml small">ദുരിതാശ്വാസ ക്യാമ്പുകൾ</span>
         </span>
       </a>
+    <a href="https://www.google.com/maps/d/u/0/viewer?mid=19pdXYBAk8RyaMjazX7mjJIJ9EqAyoRs5&ll=10.58818077013663%2C76.59135823100587&z=8" class="home-button card" role="button">
```
[the remark this answers is not in the document]
Comment (yeaske) [reply templates/home.html]: If you are anyway making the changes, make the bootstrap_icon "globe" for easy identification.

[the remark this answers is not in the document]
Comment (ssat335) [reply templates/home.html]: @yeaske OK updating now

**Comment by ssat335** [comment]
All good.

**Comment by remedcu** [comment]
This PR is similar to #500 #501 #506 and #566 

Which is already solved and merged in #566 

So, this can be closed.

@syam3526 @vigneshhari @biswaz @vrthra @bobinson

**Pull request closed without merging**